src/devices/machine/mb87030.h

```diff
@@ -74,9 +74,9 @@ class mb87030_device : public nscsi_device, public nscsi_slot_card_interface
 	constexpr static uint8_t SCTL_RESET_AND_DISABLE = 0x80;
 
 	constexpr static uint8_t SCMD_TERM_MODE = 0x01;
-	constexpr static uint8_t SCMD_PRG_XFER = 0x02;
-	constexpr static uint8_t SCMD_INTERCEPT_XFER = 0x04;
-	constexpr static uint8_t SCMD_RST_OUT = 0x08;
+	constexpr static uint8_t SCMD_PRG_XFER = 0x04;
+	constexpr static uint8_t SCMD_INTERCEPT_XFER = 0x08;
+	constexpr static uint8_t SCMD_RST_OUT = 0x10;
```

Comment:
Have you checked that this doesn’t break the HP9000/300 series machines that use this device?

Reply:
Before and after this change the hp9k drivers fail to boot the os from the hdd image in the software list resulting in a 'Root device busy' message and halting the boot process.
Similarly trying to install the os on a clean harddisk image results in an error when creating the filesystem.